snuba/admin/clickhouse/predefined_system_queries.py

```diff
@@ -0,0 +1,120 @@
+from dataclasses import dataclass
+from typing import Dict, Optional, Sequence, Type
+
+
+class _QueryRegistry:
```

Comment:
Is this a copy and paste over from the system_queries file or has it changed in some way? I think we should take the opportunity to clean it up - I believe some of the functionality here may deprecated now.

Reply:
addressed in https://github.com/getsentry/snuba/pull/2382